arch/arm/boot/dts/broadcom/bcm2712d0-rpi-5-b.dts

```diff
@@ -86,6 +86,10 @@
 	reg = <0x7d510700 0x1c>;
 };
 
+&vc4 {
```

Comment:
Fold this change into the bcm2712d0-rpi-5-b.dts commit (putting this commit first), if we're keeping it.